packages/documentation/src/stories/foundation/accessibility/accessibility.docs.mdx

```diff
@@ -12,6 +12,22 @@ import liveRegionBadAngularSample from './liveregion-bad-angular.sample.html?raw
   TODO: write accessibility page
 </div>
 
+The following article of the Swiss postal law is an official ruling concerning accessibility issues.
+
+<figure>
+  <blockquote class="blockquote" lang="de">
+    <q>Die Postdienste müssen so angeboten werden, dass Menschen mit Behinderungen
+sie in qualitativer, quantitativer und wirtschaftlicher Hinsicht unter vergleichbaren
+Bedingungen wie Menschen ohne Behinderungen beanspruchen können. Insbesondere müssen:<br/>
+a. die Zugangspunkte den Bedürfnissen von Kundinnen und Kunden mit sensorischen oder Bewegungsbehinderungen entsprechen;<br/>
+b. Sendungen von und für Menschen mit Sehbehinderungen portofrei versandt
+werden können.</q>
+  </blockquote>
+  <figcaption class="blockquote-footer">
+    <cite>§14.7 <a href="https://fedlex.data.admin.ch/filestore/fedlex.data.admin.ch/eli/cc/2012/585/20121001/de/pdf-a/fedlex-data-admin-ch-eli-cc-2012-585-20121001-de-pdf-a.pdf">Swiss postal law</a></cite>
```

Comment:
Instead of using the PDF, you could use the HTML with a direct anchor: https://www.fedlex.admin.ch/eli/cc/2012/585/de#art_14